nixos/modules/services/mail/roundcube.nix

```diff
@@ -142,26 +148,23 @@ in
       ${cfg.extraConfig}
     '';
 
-    services.nginx = {
+    services.nginx = lib.mkIf cfg.configureNginx {
       enable = true;
       virtualHosts = {
         ${cfg.hostName} = {
-          forceSSL = mkDefault true;
-          enableACME = mkDefault true;
```

Comment:
I have mixed feelings here, what's the rationale for removing that?

These are options that can trivially be disabled (in contrast to header configurations in extraConfig) and this will probably a notable inconvenience for ~everyone else using that.

Reply:
Since not all modules with nginx support provide these parameters by default, I assumed that it would be better this way.

Reply:
While I understand this reasoning (and I don't want to open up another bottle of worms like this by adding e.g. arbitrary headers to just a bunch of modules) this will really cause an unnecessary inconvenience for everybody upgrading to the next stable nixos. Especially fun for folks who are on nixos-unstable as this can only be spotted by reading the release notes.

If we want to fix the underlying issue, this should happen in a more coordinated effort.

Reply:
Removed this changes.